themes-default/slim/views/layouts/main.mako

```diff
@@ -129,6 +130,14 @@
         <%include file="/vue-components/quality-chooser.mako"/>
         <%include file="/vue-components/language-select.mako"/>
         <script>window.routes = [];</script>
+        <script>
+            Vue.filter('formatDate', str => {
+                if (MEDUSA.config.fuzzyDating) {
+                    return timeago().format(str);
+                }
+                return (new Date(str)).toLocaleDateString();
```

Comment:
I guess this is fine for now, but this needs to use the user preference.

Reply:
What user preference I didn't know we had a format for dates, if we do is it available in the API?

Reply:
Oh is that timepreset? Something like this? `"%I:%M %p"`